packages/api/controllers/nomination.js

```diff
@@ -139,6 +140,21 @@ const updateNomination = async (req, res) => {
           sendSurveyEmail(nomination);
         }
       }
+
+      if (nomination.status === NOMINATION_STATUS.document_review) {
```

Comment:
Should this change be part of this ticket? It looks like the ticket is only for sending the email.

Reply:
The email will be send just if the nomination.status === "Document review"

Reply:
Sorry, didn't put my comment on the right line. I meant is the timestamp supposed to be part of this ticket?

Reply:
Nope, my mistake! I assumed that ii has to update the timestamped too. Sorry I didn't ask. I'll fix that.